src/ui/Tabs.js

```diff
@@ -0,0 +1,83 @@
+import { Tab as BootstrapTab, Tabs as BootstrapTabs } from 'react-bootstrap';
+import { getValueFromTheme } from './theme';
+import * as PropTypes from 'prop-types';
```

Comment:
I think the import should just be 

`import PropTypes from 'prop-types'`

Reply:
Fixed in 3528f9e